xclim/sdba/adjustment.py

```diff
@@ -256,36 +301,76 @@ class DetrendedQuantileMapping(EmpiricalQuantileMapping):
       The method to use when detrending. If an int is passed, it is understood as a PolyDetrend (polynomial detrending) degree. Defaults to 1 (linear detrending)
     extrapolation : {'constant', 'nan'}
       The type of extrapolation to use. See :py:func:`xclim.sdba.utils.extrapolate_qm` for details. Defaults to "constant".
+    normalize_sim : bool
+      If True, scaled sim is normalized using `norm_group` and then detrended using `group`.
+        The norm is broadcasted and added back on scen using `interp='nearest'`, ignoring the passed `interp`.
+      If False, scaled sim is detrended using `norm_group`.
+      This is useful on large datasets using dask, when `norm_group` is a very small division (e.g. 'time.dayofyear')
+        because normalisation is a more efficient operation than detrending for similarly sized groups.
```

Comment:
I think it is confusing that the detrending group is controlled by a parameter called `normalize_sim`.

Reply:
True. I will try to think of another more explicit name.

Reply:
With the removal of `norm_group` I decided to keep `normalize_sim` as the argument name.